package.json

```diff
@@ -6,13 +6,13 @@
   "scripts": {
     "css-dev": "nodemon --watch **/*.less --ext less --exec npm run css",
     "css": "lessc ckanext/dia_theme/less/main.less ckanext/dia_theme/fanstatic/dia_custom.css",
-    "rua": "npm update && cp -R node_modules/rua-pattern-library/dist/assets ckanext/dia_theme/public/rua/ && cp -R node_modules/rua-pattern-library/dist/fonts ckanext/dia_theme/public/rua/ && cp -R node_modules/rua-pattern-library/dist/css ckanext/dia_theme/fanstatic/rua/ && cp -R node_modules/rua-pattern-library/dist/js ckanext/dia_theme/fanstatic/rua/ && sed -i'' -e 's|../assets/|/rua/assets/|g' ckanext/dia_theme/fanstatic/rua/css/* && sed -i'' -e 's|../fonts/|/rua/fonts/|g' ckanext/dia_theme/fanstatic/rua/css/*"
+    "rua": "npm update rua-pattern-library && cp -R node_modules/rua-pattern-library/dist/assets ckanext/dia_theme/public/rua/ && cp -R node_modules/rua-pattern-library/dist/fonts ckanext/dia_theme/public/rua/ && cp -R node_modules/rua-pattern-library/dist/css ckanext/dia_theme/fanstatic/rua/ && cp -R node_modules/rua-pattern-library/dist/js ckanext/dia_theme/fanstatic/rua/ && sed -i'' -e 's|../assets/|/rua/assets/|g' ckanext/dia_theme/fanstatic/rua/css/* && sed -i'' -e 's|../fonts/|/rua/fonts/|g' ckanext/dia_theme/fanstatic/rua/css/*"
   },
   "dependencies": {
     "less": "3.9.0",
```

Comment:
I ran `npm install` with Node 18.x installed locally and ran into a bunch of issues. Given that this is only used as an asset build tool and as such doesn't really impact security, I opted to `nvm use lts/dubnium` (latest of Node 10.x) and all worked fine. I'll create a support ticket to update the build tooling here.